src/components/Map/map.js

```diff
@@ -48,22 +52,28 @@ class Map extends Component {
     this.setSingleSourceInMap();
     this.addDistanceIndicatorLayer();
     this.findClustersInMap();
-
     this.loadProviderTypeImage(typeImages);
-    this.setState({ loaded: true });
+    this.setState({loaded: true});
   };
 
-  componentDidMount() {
+
+  addClusterList = (clusterCenter, list) => {
```

Comment:
Could you move this method to above addClusterMouseOverHandlerToMapLayer and pass the element to this method rather than using getElementById? So you'd use this.addClusterList(clusterLngLat, childList, clusterEl); and can remove clusterEl.id = "clusterList";

Reply:
I moved the addClusterList method but ran into errors when I used this.addClusterList(clusterLngLat, childList, clusterEl) and removed clusterEl.id = "clusterList". I didn't write this code so I'm not clear on it's purpose.